lms/djangoapps/instructor/enrollment.py

```diff
@@ -290,27 +289,29 @@ def _reset_module_attempts(studentmodule):
 
 def _fire_score_changed_for_block(course_id, student, block, module_state_key):
     """
-    Fires a PROBLEM_SCORE_CHANGED event for the given module. The earned points are
-    always zero. We must retrieve the possible points from the XModule, as
-    noted below.
+    Fires a PROBLEM_RAW_SCORE_CHANGED event for the given module.
+    The earned points are always zero. We must retrieve the possible points
+    from the XModule, as noted below.
     """
     if block and block.has_score:
         max_score = block.max_score()
         if max_score is None:
             return
         else:
-            points_earned, points_possible = weighted_score(0, max_score, getattr(block, 'weight', None))
+            raw_earned, raw_possible = 0, max_score
     else:
-        points_earned, points_possible = 0, 0
+        raw_earned, raw_possible = 0, 0
```

Comment:
This was theoretically to cover the case where a student's state has been deleted for a problem that has since been deleted from the modulestore, but for which a grade still exists. We don't have any information about the problem anymore, but we want to be able to trace the fact that the student's grade was deleted.